content/internships/project-ideas/all/contents.lr

```diff
@@ -7,3 +7,5 @@ field:
 value:
 ---
 order: 1
+---
+filter_label: all
```

Comment:
I don't think we need these files at all if the filtering is on the frontend. They are only here because we had a separate static page for each filter.

Reply:
Removed the files as requested, kindly review again.